content/authors/dmitrypol.md

```diff
@@ -1,8 +1,8 @@
 ---
-name: Dmitry Polyakovsky
-short_name: dmitrypol
-photo: '/assets/media/authors/dmitrypol.jpeg'
-github: dmitrypol
+title: Dmitry Polyakovsky
+extra:
+    photo: '/assets/media/authors/dmitrypol.jpeg'
+    github: dmitrypol
```

Comment:
This isn't rendering correctly, it's just showing the text and not the photos.

Reply:
oh I see. It should show a posts by list(https://valkey.io/authors/kyledvs/), but this is a relatively easy fix.